dtfs-central-api/src/v1/controllers/tfm/facility/tfm-get-facilities.controller.js

```diff
@@ -23,11 +23,12 @@ exports.findFacilitiesGet = async (req, res) => {
 };
 
 exports.getAllFacilities = async (req, res) => {
-  const collection = await db.getCollection('tfm-deals');
+  const collection = await db.getCollection('tfm-facilities');
   const searchString = req.body.searchString || '';
 
   // mongodb query that returns an array of objects with the following format:
   // [{
+  //        "amendments": [amendments-array]
```

Comment:
Can this commend please be converted to multi-line instead of using single line on multiple lines please.

Reply:
done